pkg/utils/exitcodes.go

```diff
@@ -0,0 +1,11 @@
+package utils
+
+const (
```

Comment:
Didn't find any repository suitable for this...

Reply:
Ok, but we shouldn't import the node-agent in other repositories.